releasenotes/notes/cni-uds-log.yaml

```
@@ -0,0 +1,9 @@
+apiVersion: release-notes/v2
+kind: feature
+area: traffic-management
+issue:
+  - 32437
+
+releaseNotes:
+  - |
+    **Added** CNI network plugin log collection at CNI Daemonset via UDS.
```

```suggestion
    **Improved** CNI network plugin to send logs to the CNI daemonset. This allows viewing CNI logs using `kubectl logs`, instead of looking at kubelet logs.
```

Tried to make it more user-facing